tooling/redhat-camel-spring-boot-bom/pom.xml

```diff
@@ -94,6 +94,18 @@
                 <version>2.7.4</version>
             </dependency>
 
+            <!-- Override jackrabbit -->
+            <dependency>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>jackrabbit-core</artifactId>
+                <version>2.22.0</version>
```

Comment:
@cunningt I saw another PR with 2.22.1 version if I am not mistaken, will this version be updated in the next build?

Reply:
> @cunningt I saw another PR with 2.22.1 version if I am not mistaken, will this version be updated in the next build?

Yes, https://github.com/jboss-fuse/camel/pull/2819 needs to be merged first